node/src/main/scala/com/wavesplatform/api/common/AddressPortfolio.scala

```diff
@@ -100,18 +99,20 @@ class BalanceIterator(
     address: Address,
     underlying: Iterator[Seq[(IssuedAsset, Long)]],
     includeAsset: IssuedAsset => Boolean,
-    private var pendingOverrides: Map[(Address, Asset), Long]
+    private var pendingOverrides: Map[(Address, Asset), Long],
```

Comment:
I suggest using `VectorMap` here, because when I look at this function, I see:
1) Strict ordering is not required.
2) We are iterating by entries of this map <-- looks like a bug.
3) The result returned to a user.

Also we are passing `VectorMap` anyway. So this is only a "documentation" moment.

Reply:
Done